pkg/apis/core/types.go

```diff
@@ -2997,6 +2997,16 @@ type PodSpec struct {
 	// - spec.containers[*].securityContext.runAsGroup
 	// +optional
 	OS *PodOS
+	// Each scheduling gate represents a particular scenario the scheduling is blocked upon.
+	// Scheduling is triggered only when SchedulingGates is empty.
+	// In the future, we may impose permission mechanics to restrict which controller can mutate
+	// which scheduling gate. It's dependent on a yet-to-be-implemented fined-grained
+	// permission (https://docs.google.com/document/d/11g9nnoRFcOoeNJDUGAWjlKthowEVM3YGrJA3gLzhpf4)
+	// and needs to be consistent with how finalizes/liens work today.
+	//
+	// This is an alpha-leve feature enabled by feature gate PodSchedulingReadiness feature gate.
```

Comment:
typo: “level”

Reply:
Also, you only need to say “feature gate” once

Reply:
done. thanks.